<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>401</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>  ID del producto:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_id"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>  Nomb. del producto:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_producto"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>  Cantidad:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="le_cantidad"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="le_nombre"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>  Nomb. del cliente</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_entrega">
      <property name="text">
       <string>Entregar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>3</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Menu de Entrega</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
